src/execution_engines/test/security/node_js/network_attacks.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { expect } from 'chai'
+import NodeEngine from '../../../node_js'
+import { ExecutionEngines } from '../../../ExecutionEngine'
+
+describe('Puppeteer Network Security', () => {
+  beforeEach(() => {
+    process.env.ENGINE = ExecutionEngines.nodejs
+  })
+
+  describe('GET', () => {
```

Comment:
I'd vote for putting this into an array and looping given the description is the same for each